README.md

```diff
@@ -12,344 +14,141 @@
-  ```
-
-  - generate the json string static keys in a dart class
-
-    ```json
-    {
-      "msg_named": "{} مكتوبة باللغة {lang}",
-    }
-    ```
-
-    ```cmd
-    flutter pub run easy_localization:generate  -f keys -o locale_keys.g.dart
-    ```
-
-    ```dart
-    abstract class  LocaleKeys {
-      static const msg_named = 'msg_named';
-    }
-    ```
-
-    ```dart
-    Text(LocaleKeys.msg_named).tr(namedArgs: {'lang': 'Dart'}, args: ['Easy localization']),
-    ```
-
-  - generate the json Loader in a dart class
-  
-    ```cmd
-    flutter pub run easy_localization:generate
-    ```
-
-- fixed many issues.
-- Added named arguments.
-
-  ```json
-  "msg_named": "{} are written in the {lang} language",
-  ```
-
-  ```dart
-  Text(LocaleKeys.msg_named).tr(namedArgs: {'lang': 'Dart'}, args: ['Easy localization']),
-  ```
-
-### [2.1.0]
-
-- Added Error widget.
-- fixed many issues.
-- Adopted Bloc architecture.
+## Features
 
-### [2.0.2]
+- Easy translations for many languages
+- Load translations as JSON, CSV, Yaml, Xml using [easy\_localization\_loader](https://github.com/aissat/easy_localization_loader)
+- React and persist to locale changes
+- Supports `plural`, `gender`, nesting, RTL locales and more
+- Optional code generation
+- Error widget for missing translations
+- Extension methods on `Text`
+- Uses BLoC pattern
 
-- fixed many issues.
-- optimized and clean code for more stability.
+See [example](example) and [CHANGELOG.md](CHANGELOG.md).
 
-### [2.0.1]
+## Getting started
 
-- Added change locale dynamically `saveLocale` default value `true`.
-- fixed many issues.
-
-## Getting Started
-
-### Configuration
-
-Add this to your package's pubspec.yaml file:
+Add to your `pubspec.yaml`:
 
 ```yaml
 dependencies:
-  # stable version install from https://pub.dev/packages
-  easy_localization: <last_version>
-
-  # Dev version install from git REPO
   easy_localization:
-    git: https://github.com/aissat/easy_localization.git
 
 ```
 
-#### Load translations from local assets
+Add translation files as local assets to `path`, e.g:
 
-You must create a folder in your project's root: the `path`. Some examples:
+```
+/assets/translations
+```
 
-> /assets/"langs" , "i18n", "locale" or anyname ...
->
-> /resources/"langs" , "i18n", "locale" or anyname ...
+Add translation files like this
 
-Inside this folder, must put the _json_ or _csv_ files containing the translated keys:
+```
+{path}/{languageCode}.{ext}
+{path}/{languageCode}-{countryCode}.{ext}
+```
 
-> `path`/${languageCode}-${countryCode}.${formatFile}
+Example:
 
-[example:](https://github.com/aissat/easy_localization/tree/master/example)
+```
+/assets/translations/en.json
+/assets/translations/en-US.json
+/assets/translations/nb.json
+```
 
-- en.json or en-US.json
-- ar.json or ar-DZ.json
-- langs.csv
+or as a single CVS file ( or other formats ) using [easy_localization_loader](https://github.com/aissat/easy_localization_loader)
```

Comment:
this part, I think should talk about custom assets loaders for` easy_localization_loader`

Reply:
I can make a note of it, but i dont think you should need to talk about `easy_localization_loader` int `easy_localization`

Reply:
Check out latest commit